extensions/eclipse-che-theia-plugin-ext/src/browser/style/che-plugins.css

```diff
@@ -80,23 +80,23 @@
 
 .che-plugin-control-panel .menu-button {
     cursor: pointer;
-    color: var(--theia-ui-font-color2);
+    color: var(--theia-descriptionForeground);
 }
 
 .che-plugin-control-panel .menu-button:hover {
-    color: var(--theia-ui-font-color1);
+    color: var(--theia-button-foreground);
 }
 
 .che-plugin-control-panel .menu-button-active {
-    color: var(--theia-ui-font-color1);
+    color: var(--theia-button-foreground);
     border: none;
-    background: var(--theia-menu-color1);
+    background: var(--theia-menubar-selectionBackground);
     box-shadow: 0px 0px 6px rgba(0, 0, 0, 0.2);
 }
 
 .che-plugin-control-panel .menu-button-disabled {
     outline: none;
-    color: var(--theia-ui-font-color3);
+    color: var(--theia-badge-background);
```

Comment:
color property sets a text color. Could you pick another variable, not --theia-successBackground ?

Reply:
Sure, just choose any that works best for you )
https://code.visualstudio.com/api/references/theme-color